sdk/src/samples/config_devnet.json

```diff
@@ -0,0 +1,543 @@
+{
+	"app": {
+		"label": "devnet-alpha-sdk",
+		"minVersion": "0.0.0",
+		"version": "0.0.0",
+		"protocolVersion": "1.1",
+		"ipc": {
+			"enabled": true
+		}
+	},
+	"components": {
+		"logger": {
+			"fileLogLevel": "debug",
+			"logFileName": "logs/devnet/lisk.log",
+			"consoleLogLevel": "info"
+		},
+		"storage": {
+			"logFileName": "logs/devnet/lisk_db.log",
+			"host": "localhost",
+			"port": 5432,
+			"database": "lisk_dev",
+			"user": "lisk",
+			"password": "password",
```

Comment:
Suggest not to set username/password here. So while running the sample, user can use their own PostgreSQL user, rather than creating a new user. 

If usage of this file is intended for internal demo, then ok.

Reply:
That password was "always" present here - the default `lisk` user set up is described in the Postgres installation manual.

Reply:
I prefer to leave it this way, so there is a default setting.